src/types/Error.ts

```diff
@@ -0,0 +1,8 @@
+export enum Error {
+  None,
+  Add = 'add',
+  Delete = 'delete',
+  Update = 'update',
+  Load = 'load',
+  EmptyInput = 'EmptyInput',
```

Comment:
Enum values should be in PascalCase. Rename 'EmptyInput' to 'Empty_Input'.

Reply:
'Empty_Input'   -   it's not a Snake Case ?!